### PR TITLE
Solves #15

### PR DESCRIPTION
**Pull request by mgrundkoetter**

### CODE REVIEW

File: ext_emconf.php [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
+    'category'         => 'plugin',
+    'author'           => 'Mittwald CM Service',
+    'author_email'     => 'support@mittwald.de',
+    'author_company'   => 'Mittwald CM Service',
+    'dependencies'     => 'cms,extbase,fluid,sr_feuser_register,static_info_tables',
+    'state'            => 'stable',
+    'uploadfolder'     => true,
+    'createDirs'       => 'typo3temp/typo3_forum,typo3temp/typo3_forum/gravatar,uploads/tx_typo3forum/workflowstatus,uploads/tx_typo3forum/attachements',
+    'modify_tables'    => 'fe_users',
+    'clearCacheOnLoad' => false,
```
Comment (DavidBruchmann) [inline ext_emconf.php:13]: Lining up thing vertically is fine with me for readability!
Why you removed it in the other commit in the annotations?
Give us the requested time for review.
Thanks!!!

Comment (mgrundkoetter) [reply ext_emconf.php:13]: I just used auto format in PHPStorm for this one, maybe I didn't in the other one :-)

Comment (DavidBruchmann) [reply ext_emconf.php:13]: don't mind ;-)

File: ext_emconf.php [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
-    'createDirs' => 'typo3temp/typo3_forum,typo3temp/typo3_forum/gravatar',
-    'modify_tables' => 'fe_users',
-    'clearCacheOnLoad' => 0,
+    'title'            => 'typo3_forum',
+    'description'      => 'Forum extension',
+    'category'         => 'plugin',
+    'author'           => 'Mittwald CM Service',
+    'author_email'     => 'support@mittwald.de',
+    'author_company'   => 'Mittwald CM Service',
+    'dependencies'     => 'cms,extbase,fluid,sr_feuser_register,static_info_tables',
```
Comment (schams-net) [inline ext_emconf.php]: Key `dependencies` is deprecated and should not be used anymore (see [TYPO3 CoreApiReference](https://docs.typo3.org/typo3cms/CoreApiReference/8.7/ExtensionArchitecture/DeclarationFile/) for v8).  
I suggest we drop this line.

File: ext_emconf.php [hunk 3]
```diff
@@ -1,31 +1,31 @@
...
+    'state'            => 'stable',
+    'uploadfolder'     => true,
+    'createDirs'       => 'typo3temp/typo3_forum,typo3temp/typo3_forum/gravatar,uploads/tx_typo3forum/workflowstatus,uploads/tx_typo3forum/attachements',
+    'modify_tables'    => 'fe_users',
```
Comment (schams-net) [inline ext_emconf.php]: Key `modify_tables` is deprecated and should not be used anymore (see [TYPO3 CoreApiReference](https://docs.typo3.org/typo3cms/CoreApiReference/8.7/ExtensionArchitecture/DeclarationFile/) for v8).  
I suggest we drop this line.

**Review by schams-net — CHANGES_REQUESTED** [review]
Thanks for your update @mgrundkoetter  
If you agree to drop these two lines, can you update your PR and I will merge it straight away?

File: ext_emconf.php (hunk 3 above)
Comment (mgrundkoetter) [reply ext_emconf.php]: has been removed

File: ext_emconf.php (hunk 2 above)
Comment (mgrundkoetter) [reply ext_emconf.php]: has been removed

**Pull request merged**